src/plugin/plugin.ts

```diff
@@ -562,6 +601,20 @@ export interface UpgradeInfo {
   readonly prorationMode?: PRORATION_MODE;
 }
 
+/**
+ * Holds the information used when upgrading from another sku. For Android use only.
+ */
+export interface GoogleProductChangeInfo {
```

Comment:
I'm not sure if we should include the store name in the models... It's true this is Google Play-only at the moment, but since this is a hybrid, I think it's generally better to make it more generic... I understand we have already added these models in other hybrids though, and there are no plans to add something similar in iOS for now. So if they add it we can modify it in the future.

Reply:
Okay, I can make that change! I think that makes enough sense to remove platform 🤷‍♂️ 

I think the original idea behind this was for cross developers to realize that this was a thing _specifically_ for Google by the name so they wouldn't try to use this on other platforms but that might just be an over engineering though 😅